Source/Wit/Private/Voice/Matcher/VoiceIntentWithEntitiesMatcher.cpp

```diff
@@ -59,6 +59,7 @@ void UVoiceIntentWithEntitiesMatcher::OnWitResponse(const bool bIsSuccessful, co
 	// have multiple entities to match
 
 	TArray<FWitEntity> MatchingEntities;
+	int8 MatchingEntitiesNum = 0;
```

Comment:
Any reason this is an int8? This effectively limits things to 127 entities which while it's unlikely to ever be exceeded it seems that there's no reason for it not to just be an int32